src/detect-replace.c

```diff
@@ -613,19 +620,16 @@ static int DetectReplaceParseTest01(void)
     de_ctx->sig_list = SigInit(de_ctx,
                                "alert udp any any -> any any "
                                "(msg:\"test\"; content:\"doh\"; replace:\"; sid:238012;)");
-    if (de_ctx->sig_list != NULL) {
-        result = 0;
-        goto end;
-    }
+    FAIL_IF_NOT_NULL(de_ctx->sig_list);
```

Comment:
is it ok to use the macro here? I believe the code after the end label won't run.